### PR TITLE
demo: proto changes with explicit routing headers for bigtable, spann…

### PR DESCRIPTION
**Pull request by viacheslav-rostovtsev**
…er, firestore

### CODE REVIEW

**Comment by 0338383620** [comment]
> …er, firestore

File: google/api/routing.proto [hunk 1]
```diff
@@ -0,0 +1,347 @@
+/*
+ This proto is here only until cl/378772749 is merged
```
Comment (chanseokoh) [inline google/api/routing.proto:2]: cl/397839221 is merged, and `rounting.proto` is already present in the repo. Closing.

Comment (chanseokoh) [reply google/api/routing.proto:2]: Actually, I don't have permissions to close this. @viacheslav-rostovtsev will you close this?

File: google/api/routing.proto
Comment (Jorgeadrianmelgar) [inline google/api/routing.proto:1]: 672

File: google/api/routing.proto [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+message Routing {
+  // A collection of Header Parameter specifications
+  // **NOTE:** If multiple parameters have the same name, "last one wins" rule
+  // is used to determine which one gets sent
+  repeated RoutingParameter routing_parameters = 1;
+}
+
+// A projection from an input message to the GRPC or REST header
+message RoutingParameter {
+  // A field to extract the header key-value pair from
```
Comment (Jorgeadrianmelgar) [inline google/api/routing.proto:329]: Y

**Review by Jorgeadrianmelgar — CHANGES_REQUESTED** [review]

File: google/api/BUILD.bazel [hunk 3]
```diff
@@ -181,6 +181,14 @@ proto_library(
...
+proto_library(
+    name = "routing_proto",
+    srcs = ["routing.proto"],
+    deps = [
+        "@com_google_protobuf//:descriptor_proto",
+    ],
+)
+
 proto_library(
     name = "service_proto",
```
Comment (Jorgeadrianmelgar) [inline google/api/BUILD.bazel:193]: @jorge

**Pull request closed without merging**